### PR TITLE
Fix Flarum logo in README.md

### PR DESCRIPTION
**Pull request by gomzyakov**
**Changes proposed in this pull request:**

- Fix Flarum logo in `README.md`

**Screenshot**

Before fix:

image

After fix:

image

### CODE REVIEW

**Comment by luceos** [comment]
Hi,

Would be better if it linked to the svg.

See https://github.com/flarum/framework/commit/a0025719841dfbd004a6044e4894a4a912a7f877

Thanks.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <p align="center">
-<a href="https://flarum.org/"><img src="https://flarum.org/assets/img/logo.png"></a>
+<a href="https://flarum.org/"><img src="https://discuss.flarum.org/assets/logo-2q2rcevl.png"></a>
```
Comment (gomzyakov) [inline README.md:2]:
```suggestion
<a href="https://flarum.org/"><img src="https://flarum.org/images/flarum.svg"></a>
```

**Comment by gomzyakov** [comment]
> Would be better if it linked to the svg.

@luceos Hello!

Thanks for the tip) I replaced the link with SVG:

image

**Comment by luceos** [comment]
Thank you! 🙏

**Pull request merged**